.github/workflows/testnet-release.yml

```diff
@@ -0,0 +1,33 @@
+name: "Testnet Community Release"
+
+on:
+  push: 
+    branches: 
+    - main
+
+env:
+  CHAIN: "dali"
+  RELEASE_VERSION: "latest"
+
+jobs:
+  build-and-publish:
+    runs-on: 
+    - self-hosted
+    - linux
+    - x64
+    - sre
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: 0
+        ref: main
+
+    - name: Build and Push Artifacts to gcloud
```

Comment:
i am not sure, so we may have markdown table of deploymentrs. 
currenc script does not feels right for me